File: Ruby/railroad2.rb
=begin
 *
 *	Problem:		Railroad
 *	Problem ID:	railroad2
 *	url:				https://open.kattis.com/problems/railroad2
 *	Solution:		Ruby
 *
=end

class Solver

	def solve
		input = gets.chomp.split(" ")
		number_junctions, number_switches = input.map(&:to_i)
		if (number_switches % 2 == 0)
			puts "possible"
		else
			puts "impossible"
		end
	end
	
end

solver = Solver.new
solver.solve

File: Java/railroad2.java
/*
 *	Problem: "Railroad"
 */

import java.util.Scanner;

public class railroad2 {
	
	/*-----main-----*/
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int x = sc.nextInt();
		int y = sc.nextInt();
		
		if (y % 2 == 0)
			System.out.println("possible");
		else
			System.out.println("impossible");	
	}
}